pattern-library/html/boilerplate-header+footer.html

```diff
@@ -0,0 +1,86 @@
+<!--
+edX Pattern Library: Boilerplate HTML Template: Header + Footer
+
+NOTES:
+* base template with the additon of how to handle an app-centric header and footer semantically.
+* this assumes that the header and footer need to have their own visual band/are full width against the viewport. If that's not needed, the ``wrapper-*`` elements around them can be removed in favor of the ``wrapper-view`` grid/layout/visual control.
+-->
+
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head>
+        <meta charset="utf-8">
+        <!-- always force latest IE rendering engine or request chrome rrame -->
+        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
```

Comment:
If you're aiming for strict validation here, it's best to define these in the headers for the document, not in the page source itself.  IE=edge,chrome=1 is not a valid value for the content attribute here.